tests/test_bgp_af.rb

```diff
@@ -456,23 +604,38 @@ def test_dampening
     ############################################
     # Set and verify 'dampening' with defaults #
     ############################################
-    vrf = 'orange'
+    if platform == :nexus
+      vrf = 'orange'
+    elsif platform == :ios_xr
+      vrf = 'default'
```

Comment:
The command is not available under a named vrf in XR, only under default.  I found several commands like this.